Archives/VolFile.cpp

```diff
@@ -430,25 +429,30 @@ namespace Archives
 	{
 		m_HeaderLength = ReadTag(std::array<char, 4> { 'V', 'O', 'L', ' ' });
 
+		// Make sure the file is large enough to contain the header
+		if (archiveFileReader.Length() < m_HeaderLength + sizeof(SectionHeader) * 2) {
```

Comment:
This check was originally meant to verify minimum size was at least 1 `SectionHeader`.

Reply:
Good point. I'll fix this bug and read the check to ensure the volume is big enough for the initial 'VOL ' header as well.